configs/pinia.json

```diff
@@ -57,11 +65,25 @@
     "lvl4": ".content h4",
     "lvl5": ".content h5",
     "lvl6": ".content p, .content li",
-    "text": ".content [class^=language-]"
+    "text": ".content [class^=language-]",
+
+    "api": {
+      "lvl0": {
+        "selector": "",
+        "default_value": "API"
+      },
+      "lvl1": ".content h1",
+      "lvl2": ".content h2",
+      "lvl3": ".content h3",
+      "lvl4": ".content h4",
+      "lvl5": ".content h5",
+      "lvl6": ".content p, .content li",
+      "text": ".content [class^=language-]"
+    }
```

Comment:
Do I need to do this for all selectors? I would like to avoid duplicating all the lvl > 0 selectors since they are the same as the default ones

Reply:
Hey @posva, yes you need to duplicate them, sorry.

Also, the correct shape should be
```json
{
  "selectors": {
    "default": {
      "lvl0": "",
      // ...
    },
    "api": {
      "lvl0": "",
      // ...
    }
  }  
}

```

Reply:
Ok, thanks! It would be nice not having to duplicate them in the future. Not sure if that part of the code is open and what language it is to send a PR 😄

Reply:
It's open source (https://github.com/algolia/docsearch-scraper) and in python, but we are actually working on providing a new infrastructure to DocSearch, stay tuned! :D

Reply:
oh but can I duplicate it only once and let the default be used for all other selectors? Like the last commit?

Reply:
Yep exactly, remove the `selectors_key` for everything that will use the `default`, and only specify and duplicate it for the API for example